contracts/common/SafeERC20.sol

```diff
@@ -75,7 +75,16 @@ library SafeERC20 {
             )
 
             if gt(success, 0) {
-                ret := mload(ptr)
+                switch returndatasize
+
+                // 32 bytes returned; is valid return
+                case 0x20 {
+                    ret := mload(ptr)
+                }
+                // Else, mark call as failed
+                default {
+                    success := 0
+                }
```

Comment:
Now that we upgraded `solidity-coverage` maybe we can stop using `switch` in favor of `if-else` statements, we can tackle that in a separate PR tho and migrate all at once

Reply:
Unfortunately there's no `else` to go along with the if ([docs](https://solidity.readthedocs.io/en/v0.4.24/assembly.html#if)) :(